src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1545,42 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Creating regex mapping
+You can use **webadmin** to create regex mappings.
+In RecipientRewriteTable, there are many types of mappings, such as
```

Comment:
This sentence `In RecipientRewriteTable, there are many types of mappings, such as `address`, `alias`,... The `regex` is one of them.` is a totology, do not add value and thus should be removed